pkgs/by-name/id/idmail/package.nix

```diff
@@ -38,14 +40,18 @@ rustPlatform.buildRustPackage rec {
   RUSTC_BOOTSTRAP = 1;
   RUSTFLAGS = "--cfg=web_sys_unstable_apis";
 
+  # Fix for Tailwindcss v3 to use SSL on darwin during cargo leptos build
+  OPENSSL_CONF = lib.optionalString stdenv.isDarwin "${openssl.out}/etc/ssl/openssl.cnf";
```

Comment:
I still find this very strange, since it shouldn't be using openssl at all on macos, instead the Security framework. cc @wegank for ideas.